### ISSUE TITLE
Possibly PING frame rate limiting error due to ping_frame_count not cleared

### ISSUE THREAD

**Issue by shigeki** [apache/trafficserver#5832]
After applying HTTP/2 rate limiting for HTTP/2 DoS protection, we have observed several PING rate limits errors as
```
 ERROR: HTTP/2 connection error client_ip=xxx.xxx.xxx.xxx session_id=xxxxxxxxxx recv ping too frequent PING frame
```
Our `max_ping_frames_per_minute` is a default value of 60. 

The errors were occurred from Chrome accesses. Chrome seems to send PING frame after some idle time as https://chromium.googlesource.com/chromium/src/+/refs/heads/master/net/spdy/spdy_session.cc#2615. 

We could not identified if this is a real PING flood. Possibly,  the error might be caused by the `ping_frame_count` not to be cleared when PING is received in 60 secs without changing `counter_index` like  
```
setInterval(function() {
  let now = Math.floor(Date.now()/1000) % 60;
  let index = (now >= 30) - 0;
  if (index == 0) {
    doPing();
  }
}, 1000);
```

**Comment by masaori335** [comment]
I agree with the counter logic has bug.
```
[Aug 16 15:56:49.446] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 1
[Aug 16 15:56:59.836] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 2
[Aug 16 15:57:30.539] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 3
[Aug 16 15:57:55.164] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 4
[Aug 16 15:58:32.169] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 5
[Aug 16 15:58:42.651] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 6
[Aug 16 15:58:53.002] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 7
[Aug 16 15:59:31.425] [ET_NET 1] DEBUG: <Http2ConnectionState.cc:1960 (increment_received_ping_frame_count)> (http2_ping) ping_frame_count: 0 8
```

**Comment by shigeki** [comment]
The issue was resolved by #5833. Thanks.
